src/test/java/POM/HomePage.java

```diff
@@ -0,0 +1,33 @@
+package POM;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage {
+
+    public HomePage(WebDriver givenDriver)
+    {
+        super(givenDriver);
+    }
+
+   By playerControlsBtn = By.cssSelector("#mainFooter div[class='side player-controls']");
+
+   By playNextSongBtn = By.cssSelector("#mainFooter span[class='play']");
+
+   By soundBar = By.cssSelector("img[alt='Sound bars']");
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```